locale/fa_IR/emailTemplates.xml

```diff
@@ -0,0 +1,1105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../lib/pkp/dtd/emailTemplateData.dtd">
+
+<!--
+  * locale/fa_IR/emailTemplates.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
Copyrights should be to Simon Fraser University, not Simon Fraser University Library.